test/specs/commonTests.js

```diff
@@ -893,21 +893,13 @@ export const implementsTextAlignProp = (Component, options = {}) => {
  * @param {Object} [options.requiredProps={}] Props required to render the component.
  */
 export const implementsVerticalAlignProp = (Component, options = {}) => {
-  const { requiredProps = {} } = options
   const { assertRequired } = commonTestHelpers('implementsVerticalAlignProp', Component)
 
   describe('verticalAlign (common)', () => {
     assertRequired(Component, 'a `Component`')
 
     _noDefaultClassNameFromProp(Component, 'verticalAlign', options)
     _noClassNameFromBoolProps(Component, 'verticalAlign', options)
-
-    _.each(Component._meta.props.verticalAlign, (propVal) => {
```

Comment:
Let's implement a similar solution as we will do in https://github.com/Semantic-Org/Semantic-UI-React/pull/1165#discussion_r96260999